packages/atomic/src/components/search/facets/facet-header/facet-header.tsx

```diff
@@ -54,6 +54,7 @@ export const FacetHeader: FunctionalComponent<FacetHeaderProps> = (props) => {
         part="clear-button"
         class="flex items-baseline max-w-full p-2 text-sm"
         title={clearFiltersForFacet}
+        ariaLabel={clearFiltersForFacet}
```

Comment:
aria label + title, won't screen readers repeat both?

Reply:
Yeah I vaguely remember problems about that in JSUI.

Reply:
I do recall that.

In this particular case, I'll remove the `title` since I see no use for it. However, in other cases, I checked and of the 5 screen readers we support, only TalkBack seems to redundantly read both, so I don't think it's too bad to give both of them the same value.